src/gui/sdlmain.cpp

```diff
@@ -1315,23 +1315,6 @@ static SDL_Window* SetWindowMode(const RenderingBackend rendering_backend,
 			flags |= SDL_WINDOW_OPENGL;
 		}
 
-		// We need a context to query the vendor string.
-		const auto temp_window = SDL_CreateWindow(
-		        "", 0, 0, 200, 200, SDL_WINDOW_OPENGL | SDL_WINDOW_HIDDEN);
-		if (temp_window == nullptr) {
-			LOG_ERR("SDL: Failed to create temporary window: %s",
-			        SDL_GetError());
-			return nullptr;
-		}
-		const auto temp_context = SDL_GL_CreateContext(temp_window);
-		if (temp_context == nullptr) {
-			LOG_ERR("OPENGL: Failed to create temporary context: %s",
-			        SDL_GetError());
-			return nullptr;
-		}
-
-		SDL_GL_DeleteContext(temp_context);
-		SDL_DestroyWindow(temp_window);
```

Comment:
Yeah AFAIK this is the "standard" solution to query the vendor string if you want to initialise your real context differently depending on the vendor.

Maybe refactor into a function called `get_opengl_vendor`? Could save some dicking around later if we need it. But then we might just migrate to the new SDL3 shader thing and not use OpenGL directly anymore.

I'm 50/50, I'll leave it to you.

Reply:
I don't understand your comment. I'm removing this vendor querying code because we're not using it. We were previously checking for "Intel" because at some point Intel drivers had problems with SRGB. That was removed in 3d7852b71e0d2874057162025bd477f8403b4f32 because I guess newer drivers don't have that problem.

Reply:
Yeah, I was thinking we'd maybe need to check vendor strings again in 2 months... for reasons 😄 But maybe don't leave code there "just in case", agreed. I get it that currently it's unused code.

Reply:
Oh, I see what you're saying now. The code I removed doesn't even have the part that gets the vendor string. That part was removed earlier. All that's left is creating and destroying a temp window and context. It's not even a fully functional block if we wanted to refactor :laughing:

Reply:
Right, I just skimmed it. In that case, remove it with _extreme prejudice_ 😎 🤘🏻